alfred-telemetry-platform/src/test/java/eu/xenit/alfred/telemetry/binder/NodeMetricsTest.java

```diff
@@ -0,0 +1,75 @@
+package eu.xenit.alfred.telemetry.binder;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import org.alfresco.repo.domain.node.NodeDAO;
+import org.alfresco.repo.domain.permissions.AclDAO;
+import org.alfresco.repo.transaction.RetryingTransactionHelper;
+import org.alfresco.service.transaction.TransactionService;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import static org.hamcrest.Matchers.is;
+
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.when;
+
+public class NodeMetricsTest {
+
+    @Test
+    public void testNodeMetrics(){
+        MeterRegistry meterRegistry = new SimpleMeterRegistry();
+        NodeDAO nodeDAO = Mockito.mock(NodeDAO.class);
+        AclDAO aclDAO = Mockito.mock(AclDAO.class);
+        TransactionService transactionService = createTransactionService();
+        NodeMetrics nodeMetrics = new NodeMetrics(nodeDAO, aclDAO, transactionService);
+        nodeMetrics.bindTo(meterRegistry);
+        when(nodeDAO.getMaxNodeId()).thenReturn((long) 1.0);
```

Comment:
preparing the mock is usually done where the mock is declared, not when it's actually used